### PR TITLE
Refact only | Optimization

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Logic/CommandLineConvert.cs [hunk 1]
```diff
@@ -610,10 +614,13 @@ internal static bool BatchConvertSave(string toFormat, string offset, Encoding t
                                         s += "_" + className + format.Extension;
                                     outputFileName = FormatOutputFileNameForBatchConvert(s, sf.Extension, outputFolder, overwrite);
                                     File.WriteAllText(outputFileName, newSub.ToText(sf), targetEncoding);
+                                    exitLoop = true;
                                 }
                             }
                         }
                         Console.WriteLine(" done.");
+                        if (exitLoop)
+                            break;
```
Comment (ivandrofly) [inline src/Logic/CommandLineConvert.cs]: Don't keep loop once File is already written on disk

Comment (niksedk) [inline src/Logic/CommandLineConvert.cs]: Cool :) 
I don't think you need the "exitLoop" variable, right?

Comment (ivandrofly) [inline src/Logic/CommandLineConvert.cs]: yeah, the reason I use a variable is that there is one that is located inside another for so... what you think?!

Comment (niksedk) [inline src/Logic/CommandLineConvert.cs]: A "break;" after 'Console.WriteLine(" done.");' should be fine.

Comment (ivandrofly) [inline src/Logic/CommandLineConvert.cs]: :+1:

Comment (ivandrofly) [inline src/Logic/CommandLineConvert.cs]: :+1:

**Pull request closed without merging**